### ISSUE TITLE
Chatbot QnA feature for given text corpus

### ISSUE THREAD

**Issue by shashankMadan-designEsthetics** [huggingface/transformers#3648]
Is there a way we can have a chatbot specifically which can answer questions on a given text corpus?
Is there any transformer model which we can train for this and how?

**Comment by parmarsuraj99** [comment]
You might want to check this, It has a very nice example. [https://huggingface.co/transformers/model_doc/bert.html#bertforquestionanswering](https://huggingface.co/transformers/model_doc/bert.html#bertforquestionanswering)

**Comment by shashankMadan-designEsthetics** [comment]
> You might want to check this, It has a very nice example. https://huggingface.co/transformers/model_doc/bert.html#bertforquestionanswering

Well, I've known this solution for quite some time, this give outputs of start and end logits.
But i'd like to know if there is any implementation like if i just feed in just a corpus of text say a article, essay and so on and ask questions it would give some relevant outout
